src/js_native_api_v8.cc

```diff
@@ -468,12 +468,14 @@ class Reference : public RefBase {
       // the reference itself has already been deleted so nothing to do
       return;
```

Comment:
It is this branch that early breaks from the SecondPassCallback. However, it seems that we did fail to reach to this branch.